src/css/styles.scss

```diff
@@ -1,46 +0,0 @@
-// Load CSS Reset from NPM
```

Comment:
This is a really important file - make sure it doesn't get deleted! This is where your CSS will go when you move things into Github.